requirements.txt

```diff
@@ -1,3 +1,4 @@
+lxml
```

Comment:
pin version.
check what you've install on your environment (pip freeze) and pin it
i.e. your version is 1.2.3, write:
lxml~=1.2.3

Reply:
- [ ] pin version

Reply:
what is ~=?
shouldn't it be >= or i we not sure if a future update could break it?

Reply:
update minor and patch versions but not major.